templates/text-gen-service/README.md

```diff
@@ -0,0 +1,21 @@
+# Anyscale Service - Text generation
+
+This template provides an example of Anyscale LLM service to generate GPT2-outputs from a given prompt.
```

Comment:
nit: Should "service" be uppercased "Service" to indicate this is referring to Anyscale Services (or is this meant to be referring to a generic service)? Same for everywhere else it's mentioned.